warehouse/query-core/src/test/java/datawave/test/MacTestUtil.java

```diff
@@ -0,0 +1,166 @@
+package datawave.test;
+
+import static org.junit.jupiter.api.Assertions.fail;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.accumulo.core.client.AccumuloException;
+import org.apache.accumulo.core.client.AccumuloSecurityException;
+import org.apache.accumulo.core.client.TableExistsException;
+import org.apache.accumulo.core.client.TableNotFoundException;
+import org.apache.accumulo.core.client.admin.TableOperations;
+import org.apache.accumulo.minicluster.MiniAccumuloCluster;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * A collection of useful utilities for tests that rely on {@link MiniAccumuloCluster}.
+ */
+public class MacTestUtil {
+
+    private static final Logger log = LoggerFactory.getLogger(MacTestUtil.class);
+
+    private MacTestUtil() {
+        // enforce static access
+    }
+
+    /**
+     * Create or recreate the given table. This ensures that each test method has a fresh table.
+     *
+     * @param tops
+     *            the {@link TableOperations}
+     * @param tableName
+     *            the table name
+     */
+    public static void createOrRecreate(TableOperations tops, String tableName) {
+        try {
+            if (tops.exists(tableName)) {
+                tops.delete(tableName);
+            }
+            tops.create(tableName);
+        } catch (AccumuloException | AccumuloSecurityException | TableNotFoundException | TableExistsException e) {
+            fail("Failed to delete/create table");
```

Comment:
I think `fail()` will throw an AssertException or something like that so the RuntimeExceptions will never be reached. But given that this utility isn't about assertions - it's more about shortcuts for Accumulo - I would just throw the runtime exceptions here and remove all the `fail()`s.